content/nextflow-on-aws-batch/40_nextflow202/37_batched-squared-debug.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Batch Sqared Debug"
+chapter: false
+weight: 37
+---
+
+
+If you want to debug what is going on you may start a container similar to what is started within the `workflow-queue`.
+
+```bash
+docker run -ti --rm -e AWS_REGION=${AWS_REGION} \
+                    -e NF_SCRIPT=script7.nf \
+                    -e BUCKET_NAME_RESULTS=${BUCKET_NAME_RESULTS} \
+                    -e AWS_BATCH_JOB_ID=${RANDOM} \
+                    ${REPO_URI}:${IMG_TAG}:2020-04-24.1 bash
```

Comment:
This bit is not clear. First there is an error in the image reference

```
docker run -ti --rm -e AWS_REGION=${AWS_REGION} \
                    -e NF_SCRIPT=script7.nf \
                    -e BUCKET_NAME_RESULTS=${BUCKET_NAME_RESULTS} \
                    -e AWS_BATCH_JOB_ID=${RANDOM} \
                    ${REPO_URI}:${IMG_TAG} bash
```

Second is difficult to infer that once in the bash you have to execute the entrypoint.sh for people that don't understand well how docker works. 

The next entry looks more as what should have happen instead.

Reply:
yeah - let's see how this is dealt with. IMHO it makes more sense to let the attendees use a bash shell instead of hide everything by executing directly.